locations/storefinders/store_locator_plus_self.py

```diff
@@ -38,10 +40,27 @@
 # location).
 
 
-class StoreLocatorPlusSelfSpider(Spider):
-    searchable_points_files = []
-    search_radius = 0
-    max_results = 0
+class StoreLocatorPlusSelfSpider(Spider, AutomaticSpiderGenerator):
```

Comment:
Additional spider attributes to autodetect

- allowed_domain
- searchable_point_files (generating a placeholder would be good, even if commented out)
- max_results > 0; as you get warnings about truncation which are misleading

Compare https://github.com/alltheplaces/alltheplaces/issues/8099 to the PR that closes it

Reply:
Thanks. I also uncovered this problem in #8051 where the official storefinder page is not sending request parameters that include the search radius nor maximum number of results to return. I do agree with at least outputting commented out attributes that can't be determined automatically.

Reply:
So this is mergable, let's make this into a new issue/feature request to address post merge.